src/py/reactpy/reactpy/backend/types.py

```diff
@@ -11,11 +11,11 @@
 
 
 @runtime_checkable
-class BackendImplementation(Protocol[_App]):
+class BackendProtocol(Protocol[_App]):
```

Comment:
Given that `BackendType` is going to have at minimum 4 attributes, and possibly more in the future, should we consider changing our backends to use classes instead?

It feels really disjointed to read right now.

Reply:
Given that none of the methods require any shared state (and I'd like to avoid that if possible) I don't think we need these to be attached to classes.

Reply:
Conceptually they're still tied together though. They could all be static methods in a class perhaps?

When I was looking at creating a backend, it was fairly hideous to stare at our existing files to figure out what's going on.

Reply:
If things become more complicated we can consider doing that. For now, it seems like it would be equally complicated to make that change without breaking the ability to do `from reactpy.backend.package import configure`.